Assets/MixedRealityToolkit/EventDatum/Input/FocusEventData.cs

```diff
@@ -10,13 +10,18 @@ namespace Microsoft.MixedReality.Toolkit.Core.EventDatum.Input
     /// <summary>
     /// Describes an Input Event associated with a specific pointer's focus state change.
     /// </summary>
-    public class FocusEventData : BaseEventData
+    public class FocusEventData : BaseInputEventData
```

Comment:
What's the benefit of this change?

Reply:
The MixedRealityInputSystem's HandleEvent method casts to BaseInputEventData. I routed the focus events through it too, therefore the change.

Reply:
What's the benefit of sending focus events through the `HandleEvent` path instead of directly to the focused / unfocused object?

Reply:
the InteractionReceiver could listen to everything that happens to another gameobject, except for these.
this path uses global listeners while the previous bypassed them.